File: BiebApp/src/main/java/com/leesenleer/BiebApp/model/Boek.java
package com.leesenleer.BiebApp.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Boek {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    int id;
    String naam;
    String auteur;
    int jaarVanUitgave;
    int aantalBladzijden;
}

File: bieb-app-productie-code/angular-bieb-app/src/app/Boek.ts
import {Bieb} from "./Bieb";

export class Boek {
  id: number = 0;
  titel: string;
  auteur: string;
  jaarVanUitgave: number;
  aantalBladzijden: number;
  bieb: Bieb;

  constructor(id: number, titel: string, auteur: string, jaarVanUitgave: number, aantalBladzijden: number){
    this.id = id;
    this.titel = titel;
    this.auteur = auteur;
    this.jaarVanUitgave = jaarVanUitgave;
    this.aantalBladzijden = aantalBladzijden;
  }

}
